tests/fixtures/test_fixtures.py

```diff
@@ -172,20 +177,28 @@ def test_load_users(app, db, admin_role):
     assert current_datastore.find_user(email="user@example.com")
 
 
-def test_load_affiliations(app, db, admin_role, es_clear):
+def test_load_affiliations(
+        app, db, admin_role, es_clear, affiliations_service):
     dir_ = Path(__file__).parent
-    affiliations = AffiliationsFixture(
-        [
-            dir_ / "app_data",
-            dir_.parent.parent / "invenio_rdm_records/fixtures/data"
-        ],
-        "affiliations.yaml"
+    affiliations = SpecificVocabularyEntryWithSchemes(
+        "affiliations_service",
+        Path(__file__).parent / "app_data",
+        "affiliations",
+        {
+            "pid-type": "aff",
+            "schemes": [{
+                "id": "ROR",
+                "name": "Research Organization Registry",
+                "uri": "https://ror.org/",
+                "data-file": "vocabularies/affiliations_ror.yaml"
+            }]
+        }
     )
 
-    affiliations.load()
+    affiliations.load(system_identity, delay=False)
 
-    # app_data/users.yaml doesn't create an admin@inveniosoftware.org user
-    service = current_service_registry.get("rdm-affiliations")
-    cern = service.read(identity=system_identity, id_="01ggx4157")
+    cern = affiliations_service.read(identity=system_identity, id_="01ggx4157")
     assert cern["acronym"] == "CERN"
-    pytest.raises(PIDDoesNotExistError, service.read, "cern", system_identity)
+
+    with pytest.raises(PIDDoesNotExistError):
```

Comment:
nothing against this change... just wondering if it is a style preference or there is something to be gained from using the context descriptor.

Reply:
I find it easier to read, but in this scenario I was just copying the test structure in the tests above to make sure we were testing the same things and made the raise tests the same. What I also like about this style is that it's easier to extend. It can easily become:

```python
    with pytest.raises(PIDDoesNotExistError) as e:
        affiliations_service.read("cern", system_identity)

    # check something about the raised error
    print(e)
```